bestiary/core/api.py

```diff
@@ -76,6 +78,59 @@ def add_ecosystem(ctx, name, title=None, description=None):
     return ecosystem
 
 
+@django.db.transaction.atomic
+def add_project(ctx, ecosystem_id, name, title=None):
```

Comment:
This doesn't allow to add a project as a sub-project? Is there any reason why?

Reply:
This is part of the work I'm doing at https://github.com/chaoss/grimoirelab-bestiary/pull/43, but I can add that part to this PR.

Reply:
As you wish. You can add it as a new feature in a different PR.